I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 7, in script
    define preferences.afm_enable = False
  File "renpy/common/000namespaces.rpy", line 84, in set
    raise Exception("The define statement can not be used with the preferences namespace.")
Exception: The define statement can not be used with the preferences namespace.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 7, in script
    define preferences.afm_enable = False
  File "/Applications/renpy-8.1.3-sdk/renpy/ast.py", line 2222, in execute
    self.set()
  File "/Applications/renpy-8.1.3-sdk/renpy/ast.py", line 2240, in set
    ns.set(self.varname, value)
  File "renpy/common/000namespaces.rpy", line 84, in set
    raise Exception("The define statement can not be used with the preferences namespace.")
Exception: The define statement can not be used with the preferences namespace.

macOS-14.4.1-arm64-arm-64bit arm64
Ren'Py 8.2.3.24061702
prototyping 1.0
Tue Oct 15 00:39:36 2024
